api/TransitionEvent.json

```diff
@@ -149,10 +149,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/TransitionEvent/propertyName",
           "support": {
             "chrome": {
-              "version_added": null
+              "version_added": true
```

Comment:
Should be `"26"`.

The [commit was in January of 2013](https://chromium.googlesource.com/chromium/src/+/03e7d76301b8282dcb80ed28e6272338f0df3e80), putting it in the time frame for Chrome 26.